kuma/users/tests/test_middleware.py

```diff
@@ -24,3 +24,4 @@ def test_ban_middleware(self):
 
         resp = self.client.get('/')
         self.assertTemplateUsed(resp, 'users/user_banned.html')
+        assert parsedate(resp['Expires']) <= gmtime()
```

Comment:
I thought about it, but it seems like django is changing `Cache-Control` behavior more readily than `Expires`, so I feel like this makes the test less susceptible to upstream logic changes. And the only reason I'm testing this is to make Travis pass ...